Makie/test/plots/text.jl

```diff
@@ -144,3 +144,9 @@ end
         end
     end
 end
+
+@testset "rich text operations" begin
+    @test rich("a"; font=:bold) * "b" == rich(rich("a"; font=:bold), "b")
+    @test "b" *  rich("a"; font=:bold) == rich("b", rich("a"; font=:bold))
```

Comment:
```suggestion
    @test "b" * rich("a"; font=:bold) == rich("b", rich("a"; font=:bold))
```
Minor formatting fix